package.json

```diff
@@ -37,18 +37,20 @@
   },
   "devDependencies": {
     "@babel/core": "^7.12.0",
+    "@parcel/babel-register": "2.7.0",
+    "@parcel/babel-preset": "2.7.0",
```

Comment:
These are also dependencies of /dev/bable-preset however due to them being workspaces packages, they cannot be explicitly hoisted in the `.npmrc` and instead referenced here.

As an alternative to the `.npmrc`, I can put those packages in the top level `package.json`